services/web/server/src/simcore_service_webserver/folders/_folders_repository.py

```diff
@@ -589,7 +592,7 @@ async def get_trashed_by_primary_gid(
 
     async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
         result = await conn.execute(query)
-        row = result.first()
+        row = result.one_or_none()
```

Comment:
Q: I'd guess .first would raise if no request is present?

Reply:
no it does not.

Reply:
no it returns none